plugins-compat-tester/src/main/java/org/jenkins/tools/test/hook/SkipUIHelperPlugins.java

```diff
@@ -0,0 +1,43 @@
+package org.jenkins.tools.test.hook;
+
+import hudson.model.UpdateSite.Plugin;
+import org.jenkins.tools.test.model.TestExecutionResult;
+import org.jenkins.tools.test.model.hook.PluginCompatTesterHookBeforeCheckout;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Short circuit running any UI plugins that function as a helper methods.
+ * These are installed as "plugins" by various parts of the UI, and can't
+ * be tested through Maven.
```

Comment:
For direct exclusions we already have a mechanism, if a plugin reaches this point, it should be tested.